.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*/app.log
```

Comment:
you should use this line that covers all log files or folder where log files will be storing